### DISCUSSION TITLE
Mail adress for my Uni not correct when signing up for Student Benefits

### DISCUSSION THREAD

**Discussion by xiomatic** [community/community#18936] [GitHub Education] [answered]
So my university has different mail adresses depending on what subject you study. I study German, so my mail adress is Firstname.Lastname@germanistik.uni-giessen.de. When applying for Student Benefits, I was told that my mail adress is incorrect and that ONLY mail adresses ending in @bioinfsys.uni-giessen.de or @lehramt.uni-giessen.de are accepted. Which means only 2 out of like 50+ different mail subdomains from my Uni can access the Student Benefits program.

Suggested fix: @*.uni-giessen.de

![grafik](https://user-images.githubusercontent.com/71766127/174503181-6afcf4ea-2191-409d-9cea-0c3da3b34e6e.png)

**Comment by rafasha123** [comment]
ya

**Comment by rafasha123** [comment]
561506

**Comment by melvincwng** [comment, marked as answer]
Hi @xiomatic, please raise a ticket with GitHub Education Support to investigate the issue: https://support.github.com/contact/education

They usually respond within a few business days, depending on the amount of support tickets they receive daily.

Hope this helps 😀
